frontend/src/pages/LandingPage.js

```diff
@@ -1,19 +1,60 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 import Moment from '../components/Moment';
-import sample from '../resources/sample-moment.jpg';
-import logo from '../resources/logo.svg';
-import tai from '../resources/TaiLopez.png';
+// import sample from '../resources/sample-moment.jpg';
+// import logo from '../resources/logo.svg';
+// import tai from '../resources/TaiLopez.png';
 
 class LandingPage extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            moments: null,
+            error: null,
+        };
+    };
+
+    componentWillMount(){
+        axios.get('http://localhost:16085/api/moments').then(response => {
+            let moments = response.data.moments;
+            this.setState({
+                moments,
+            });
+        }).catch(error => {
+            console.log(error);
+            this.setState({
+                error: 'Oops! Something went wrong...'
```

Comment:
Maybe a more descriptive error, your call though

Reply:
I'd argue that the user doesn't necessarily need to know the details of the error.